code-challenges.test.js

```diff
@@ -13,55 +13,131 @@
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
 // a) Create a test with expect statements for each of the variables provided.
+//Note: On previous assesments I had pseudocode for the tests
+//we were told we didn't have to so moving forward I will not
+//psuedocode tests
+describe('fibonacci', () => {
+  it('takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence', () => {
+    const fibLength1 = 6
+    // Expected output: [1, 1, 2, 3, 5, 8]
+    const fibLength2 = 10
+    // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+    expect(fibonacci(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(fibonacci(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+
+  })
+})
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
-
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
+//create a function named fibonacci that takes a number as a parameter
+const fibonacci = (number) => {
+  //The number passed must be greater than 2
+  if (number>2){
+    //create a variable named sequence that holds two numbers [1, 1]
+    //that begins our fibonacci sequence
+    sequence=[1, 1]
```

Comment:
Logic looks great in this function. Make sure you declare your variables or JavaScript will automatically make them global.